components/common/src/command/package/install.rs

```diff
@@ -39,7 +39,6 @@ use std::fs;
 use std::path::{Path, PathBuf};
 use std::str::FromStr;
 
-use ansi_term::Colour::{Blue, Green, Yellow};
```

Comment:
This is the only place that we were leveraging the `ansi_term` crate so we can now drop it as a dependency from this crate's Cargo.toml